### PR TITLE
Credit the contributors - Issue #173

### PR DESCRIPTION
**Pull request by nicolas-trouin**
#173 

Unfortunately it adds all the other commit about the french translation #181 even if I have created a new branch. I'm not very good with git/github, so if it's a problem I can try to separate what i have done for each PR so one won't need the other to work.
Of maybe I should just push every commit on #181 , I don't know ... 🤔

### CODE REVIEW

**Comment by rugk** [comment]
Yeah, this happened, because you made your French translation changes the master branch. Usual way to use Git/GitHub is to create a new branch for each Pull Request, so you still have a "clean" master branch…

So this is only about https://github.com/rugk/offline-qr-code/pull/182/commits/b77b84b9e014b3e34d9e367ab1166ae5f31706eb?

**Comment by nicolas-trouin** [comment]
> Yeah, this happened, because you made your French translation changes the master branch. Usual way to use Git/GitHub is to create a new branch for each Pull Request, so you still have a "clean" master branch…

Yep I think that's what I should have done ^^' I'll try to remember to do it next time.


> So this is only about [b77b84b](https://github.com/rugk/offline-qr-code/commit/b77b84b9e014b3e34d9e367ab1166ae5f31706eb)?

That's right !

File: src/_locales/fr/messages.json [hunk 1]
```diff
@@ -124,6 +124,10 @@
     "description": "The text of a button that sets a new color with a sufficient contrast.",
     "hash": "9d422df9f61f2b45e272ae3451ca3ded"
   },
+  "contributorsCredit": {
+	"message": "Cette extension a été traduite en Français par Nicolas Trouin (@nicolas-trouin). Cette extension n'existerait pas sans tous les autres contributeurs.",
```
Comment (rugk) [inline src/_locales/fr/messages.json]: indentation!

**Comment by rugk** [comment]
I've merged https://github.com/rugk/offline-qr-code/pull/182, so if you want to re-create this as a clean PR, you should now be able to delete your fork on GitHub ([here](https://github.com/nicolas-trouin/offline-qr-code)) and re-create this PR. (best is likely to just copy & paste the changes locally)

Anyway, I guess I'll change the wording or so anyway, so I'll just adjust it myself then.

If you really want to solve this issue, also have a look at the new feature the localizer module offers ([see doc here](https://github.com/TinyWebEx/Localizer#keeping-child-elements-in-html)). This way, you can include links in the translation…

Anyway, first, we need a clean PR… Thus I am closing this one here for now.

**Pull request closed without merging**